tests/api/runtime_handlers/test_mpijob.py

```diff
@@ -60,15 +61,15 @@ def custom_setup(self):
 
         self.launcher_pod_labels = {
             "group-name": "kubeflow.org",
```

Comment:
what about `"group-name"` `"mpi-job-name"` and `"mpi-job-role"`?

Reply:
^

Reply:
I find these labeles only  in tests, I didn't find another place where we set it.
Because of that I didn't add it to the internal labels.